.github/workflows/build.yml

```diff
@@ -9,7 +9,15 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    uses: [actions/checkout@v3, ./.github/actions/prepare]
     steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/actions/prepare
-      - run: pnpm build
+      - id: test1
+        run: |
+          pnpm build | sed 's/^/ Build Test : '
+      - id: test2
+        run: |
+          if node ./src/index.ts ; then
```

Comment:
Did you mean `node ./src/index.js`? Running `node` on a `.ts` file without any additional configuration gives `TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts"`.

Reply:
I wasn't sure about this one, which file should we run at the time of build ? I can alter this accordingly

Reply:
`node ./src/index.js`. That's the JavaScript file created by running TypeScript (`pnpm build`).